Nodejs/Product/Nodejs/Project/GlobalModulesNode.cs

```diff
@@ -1,101 +0,0 @@
-﻿//*********************************************************//
-//    Copyright (c) Microsoft. All rights reserved.
-//    
-//    Apache 2.0 License
-//    
-//    You may obtain a copy of the License at
-//    http://www.apache.org/licenses/LICENSE-2.0
-//    
-//    Unless required by applicable law or agreed to in writing, software 
-//    distributed under the License is distributed on an "AS IS" BASIS, 
-//    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or 
-//    implied. See the License for the specific language governing 
-//    permissions and limitations under the License.
-//
-//*********************************************************//
-
-using System;
-using System.Linq;
-using Microsoft.NodejsTools.Npm;
-using Microsoft.VisualStudio;
-using Microsoft.VisualStudioTools.Project;
-
-namespace Microsoft.NodejsTools.Project {
-    internal class GlobalModulesNode : AbstractNpmNode {
```

Comment:
@mousetraps This sounds like a good idea. One concern though: the Npm management window currently depends on the global modules data and therefore on the NpmBinCommand:

![image](https://cloud.githubusercontent.com/assets/12821956/16666651/8999549c-443d-11e6-8102-6a789acfe034.png)

Removing the NpmBinCommand means we should probably also remove the logic for supporting globals in the Npm window. Does that sound correct and do we want to do this? I agree that as it stands, this change only addresses one part of the global module memory usage problem.